src/sentry/tasks/merge.py

```diff
@@ -168,8 +168,7 @@ def merge_objects(models, group, new_group, limit=1000,
     for model in models:
         if logger is not None:
             logger.info('%s.merge' % model.__name__.lower(), extra={
-                'group_id.old': group.id,
-                'group_id.new': new_group.id
+                'group_id': new_group.id
```

Comment:
Why not just say `old_group_id` and `new_group_id`. As-is, we've lost the old group id.